### PR TITLE
issue 3084

### PR DESCRIPTION
**Pull request by gfcapalbo**
solving an ensure one exception encountered by client

### CODE REVIEW

File: fetchmail_inbox/model/fetchmail_inbox.py [hunk 1]
```diff
@@ -17,6 +17,6 @@ def name_get(self):
         ])
         return [
             (this.id, mails.filtered(
-                lambda x: x.res_id == this.id).subject or _('Fetchmail inbox'))
+                lambda x: x.res_id == this.id)[0].subject or _('Fetchmail inbox'))
```
Comment (hbrunn) [inline fetchmail_inbox/model/fetchmail_inbox.py]: use slicing to protect against empty list: `[:1]`

**Review by hbrunn — CHANGES_REQUESTED** [review]

**Comment by gfcapalbo** [comment]
not using mapped , but less lines of code

**Comment by hbrunn** [comment]
and messing up lint and degrading performance radically. Use the slice

**Comment by gfcapalbo** [comment]
done

**Pull request merged**